integration/backward_compatibility_test.go

```diff
@@ -34,8 +34,12 @@ func TestBackwardCompatibilityWithChunksStorage(t *testing.T) {
 	// Start Cortex components (ingester running on previous version).
 	require.NoError(t, writeFileToSharedDir(s, cortexSchemaConfigFile, []byte(cortexSchemaConfigYaml)))
 	tableManager := e2ecortex.NewTableManager("table-manager", ChunksStorageFlags, previousVersionImage)
+	// Old table-manager doesn't expose a readiness probe, so we just check if the / returns 404
+	tableManager.SetReadinessProbe(e2e.NewReadinessProbe(e2ecortex.HTTPPort, "/", 404))
```

Comment:
Instead of exposing `e2ecortex.HTTPPort` you could just add `HTTPPort()` function to `HTTPService`. This way you can get it via `tableManager.HTTPPort()`.

Reply:
Sounds good!